kbase-extension/static/kbase/js/util/kbaseApiUtil.js

```diff
@@ -58,6 +58,7 @@ define([
             }),
         ]).then(([appSpecs, appFullInfos]) => {
             appFullInfos.forEach((info, idx) => {
+                info.tag = tag;
```

Comment:
Add the app tag (release/beta/dev) to the `full_info` part of the app spec so the tag can be seen in the bulk import cell info panel (very important, obviously!).